storm-core/src/clj/org/apache/storm/ui/core.clj

```diff
@@ -863,6 +855,7 @@
                                       (.get_eventlog_host comp-page-info)
                                       (.get_eventlog_port comp-page-info)
                                       secure?)
+       "profilingAndDebuggingCapable" (not on-windows?)
```

Comment:
Sure.  We are using this var in several places, and I did not want to change all of them to fix this.